backend/corpora/lambdas/api/v1/collection.py

```diff
@@ -25,6 +28,29 @@ def get_collection_details(collection_uuid: str):
         raise ForbiddenHTTPException()
 
 
+@db_session
+def create_collection(body: object):
+    try:
+        # get user id
+        collection_data = json.loads(body)
+        collection = Collection(
+            visibility=CollectionVisibility.PRIVATE,
+            name=collection_data.name,
+            description=collection_data.description,
+            owner=collection_data.contact_name,
+            links=collection_data.links,
+            data_submission_policy_version=collection_data.data_submission_policy_version,
+        )
+
+        return make_response(jsonify({"collection_uuid": collection.id}), 201)
+    # except NotAuthorized:
+    #     return make_response(401)
+    except Exception as e:
```

Comment:
you don't need to catch exceptions here, the errors are returned in the request body.